apps/src/templates/levelSummary/FreeResponseAIEvaluation.tsx

```diff
@@ -93,24 +94,18 @@ const FreeResponseAIEvaluation: React.FunctionComponent<
         isPending={evaluationsPending}
         studentWorkEvaluations={evaluations}
         evaluationComplete={evaluationComplete}
+        totalStudentCount={responses.length}
```

Comment:
🤔 Couldn't the total student count be higher than the responses.length if there are students who haven't responded yet?

Reply:
Fixed!  Also created a [ticket](https://codedotorg.atlassian.net/jira/software/projects/TEACH/boards/65?selectedIssue=TEACH-1759) for more accurately tracking the "no response" folks.